code/modules/mob/living/simple_animal/friendly/mouse.dm

```diff
@@ -34,13 +34,28 @@
 	holder_type = /obj/item/holder/mouse
 	can_collar = 1
 	gold_core_spawnable = CHEM_MOB_SPAWN_FRIENDLY
+	var/chew_probability = 1
 
 /mob/living/simple_animal/mouse/Initialize(mapload)
 	. = ..()
 	AddComponent(/datum/component/squeak, list('sound/creatures/mousesqueak.ogg' = 1), 100)
 
 /mob/living/simple_animal/mouse/handle_automated_speech()
 	..()
+	if(prob(chew_probability))
+		var/turf/simulated/floor/F = get_turf(src)
+		if(istype(F) && !F.intact)
+			var/obj/structure/cable/C = locate() in F
+			if(C && prob(15))
+				if(C.avail())
+					visible_message("<span class='warning'>[src] chews through [C]. It's toast!</span>")
+					playsound(src, 'sound/effects/sparks2.ogg', 100, 1)
+					C.deconstruct()

```

Comment:
I'd prefer if there were another proc defined on the mouse, `toast`, that you then call here, instead of adding another parameter to `death`, which could lead to issues with adding a general argument to `death` on the general mob type.

Reply:
seems easy enough